wespeaker/bin/extract.py

```diff
@@ -81,8 +92,24 @@ def extract(config='conf/config.yaml', **kwargs):
                                  embed_scp) as writer:
             for _, batch in tqdm(enumerate(dataloader)):
                 utts = batch['key']
-                features = batch['feat']
-                features = features.float().to(device)  # (B,T,F)
+                if frontend_type == 'fbank':
+                    features = batch['feat']
+                    features = features.float().to(device)  # (B,T,F)
+                else:  # 's3prl'
+                    wavs = batch['wav']  # (B,1,W)
```

Comment:
Is that possible to skip the fbank extraction in the dataset part if we only the the raw wave? I think the fbank extraction on CPU will take some time.